data/announcements/scheduled.yaml

```
@@ -71,4 +71,26 @@ announcements:
     <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/">KubeCon + CloudNativeCon EU 2021</a> <em>virtual</em>.
   message: |
     4 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
-    May 4 - May 7, 2021.
\ No newline at end of file
+    May 4 - May 7, 2021.
+
+- name: Kubecon 2021 NA
+  startTime: 2021-10-01T00:00:00
+  endTime: 2021-10-15T00:00:00
```

I'd leave the banner up until the end of the conference, eg:
```suggestion
  endTime: 2021-10-16T01:00:00
```

Times are in the UTC timezone